src/backend/joanie/payment/api.py

```diff
@@ -68,3 +68,17 @@ def get_queryset(self):
             else self.request.user.username
         )
         return models.CreditCard.objects.filter(owner__username=username)
+
+    @action(
+        methods=["POST"],
+        detail=False,
+    )
```

Comment:
I know we have many endpoints in this case, but we should specify `url_path` to avoid `_` in urls:
```suggestion
    @action(
        methods=["POST"],
        detail=False,
        url_path="tokenize-card",
    )
```